### PR TITLE
Feature/list component

### PR DESCRIPTION
**Pull request by josch87**
added list and listItem Components

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,22 +2,24 @@ import "./App.css";
 import Form from "./components/Form/Form.js";
 import useLocalStorageState from "use-local-storage-state";
 import { uid } from "uid";
+import List from "./components/List/List";
```
Comment (ahohnsen) [inline src/App.js:5]: I recommend sorting the imports for a better overview, e.g. first imports from react and other packages, then imports from components and so on 🙂

File: src/components/List/ListItem/ListItem.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import "./ListItem.style.css";
+
+export default function ListItem({ children }) {
+  return <li className="list__item">{children}</li>;
+}
```
Comment (ahohnsen) [inline src/components/List/ListItem/ListItem.js:5]: Of course, you can make a component for your list item, but in my opinion it is not really necessary here. As your component is only one line of code (the `li` element), you can add it directly in your `List` component.

❗ Please do not forget that every list item will need a unique key (see[ here](https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key)).

File: src/index.css [hunk 3]
```diff
@@ -1,31 +1,31 @@
 :root {
-	--background-color: #ffffff;
-	--primary-color: rgb(20, 20, 20);
-	--secondary-color: rgb(191, 191, 191);
-	--border: 1px solid var(--primary-color);
+  --background-color: #ffffff;
+  --primary-color: rgb(125, 125, 125);
+  --secondary-color: rgb(191, 191, 191);
+  --border: 1px solid #282828;
```
Comment (ahohnsen) [inline src/index.css:5]: Nice 💪🏼

**Review by ahohnsen — CHANGES_REQUESTED** [review]
Well done!

Please add a unique key for your list items before you continue :)

File: src/App.js (hunk 1 above)
Comment (vaupunkt) [reply src/App.js:5]: done that

File: src/components/List/ListItem/ListItem.js (hunk 2 above)
Comment (vaupunkt) [reply src/components/List/ListItem/ListItem.js:5]: we added the key in our other branch/feature :)

**Pull request merged**